datashader/__init__.py

```diff
@@ -14,3 +14,10 @@
     from .dask import *
 except ImportError:
     pass
+
+try:
+    import os, pytest
+    def test():
+        pytest.main(os.path.dirname(__file__))
+except ImportError:
+    pass
```

Comment:
Perhaps:

```python
def test():
    import os
    import pytest
    pytest.main(os.path.dirname(__file__))
```

That way the top level namespace isn't polluted with pytest/os imports. Not a big deal though.